scripts/jestIntegration.config.json

```diff
@@ -0,0 +1,6 @@
+{
```

Comment:
This should probably be baked into the `package.json` - you can always run jest with only a particular project if you want to narrow the scope. `jest --project integration`

Reply:
@chrislloyd indeed, that should be the case if we make puppeteer a jest project. It seems like we're running into a couple of issues though if we're going for that approach:

![image](https://user-images.githubusercontent.com/127199/40143743-1dc87938-5911-11e8-9705-6245c2f502e4.png)

```
{
  "displayName": "puppeteer",
  "preset": "jest-puppeteer",
  "testMatch": ["<rootDir>**/__integration__/**/*.integration.js"]
}
```